pkgs/development/python-modules/scikitlearn/default.nix

```diff
@@ -54,16 +54,33 @@ buildPythonPackage rec {
     joblib
     threadpoolctl
   ];
-  checkInputs = [ pytest ];
+  checkInputs = [ pytestCheckHook ];
+
+  preCheck = ''
+    export OMP_NUM_THREADS=1
+    export HOME=$(mktemp -d)
+    cd "$HOME"
+  '';
+
+  pytestFlagsArray = [
+    "--pyargs sklearn"
+    # verbose build outputs needed to debug hard-to-reproduce hydra failures
+    "-v"
+  ];
+
+  disabledTests = lib.optionals stdenv.isDarwin [
```

Comment:
I'd say these should be disabled on all platforms, not just darwin.

Reply:
Sure. I removed the darwin specific logic.